### PR TITLE
feat : Display + button to create a quick chat discussion from chat drawer - EXO-60883

### PR DESCRIPTION
**Pull request by sofyenne**
We will display `+` button in the chat drawer which will create a quick chat discussion.

### CODE REVIEW

File: application/src/main/webapp/vue-app/chatConstants.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ export const chatConstants = {
   ACTION_ROOM_OPEN_CHAT: 'exo-chat-room-open-requested',
   ACTION_FILTER_ROOM_TYPE: 'exo-chat-room-filter-changed',
   ACTION_CHAT_OPEN_DRAWER: 'exo-chat-open-drawer',
+  ACTION_CHAT_OPEN_ROOM_DRAWER: 'exo-chat-open-room-drawer',
```
Comment (azayati) [inline application/src/main/webapp/vue-app/chatConstants.js]:
```suggestion
  ACTION_CHAT_OPEN_QUICK_CREATE_DISCUSSION_DRAWER: 'exo-chat-open-quick-create-discussion-drawer',
```

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 2]
```diff
@@ -228,6 +236,8 @@ export default {
     document.addEventListener(chatConstants.ACTION_ROOM_OPEN_CHAT, this.openRoom);
     chatWebStorage.registreEventListener();
     chatWebSocket.registreEventListener();
+    this.$featureService.isFeatureEnabled('quickCreateChatDiscussion')
+      .then(enabled => this.chatDiscussionFeatureEnabled = enabled);
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]:
```suggestion
      .then(enabled => this.quickCreateChatDiscussionFeatureEnabled = enabled);
```

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 3]
```diff
@@ -156,6 +163,7 @@ export default {
       external: this.$t('exoplatform.chat.external'),
       chatLink: `/portal/${eXo.env.portal.portalName}/chat`,
       titleActionComponents: miniChatTitleActionComponents,
+      chatDiscussionFeatureEnabled: false
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]:
```suggestion
      quickCreateChatDiscussionFeatureEnabled : false 
```

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 4]
```diff
@@ -70,6 +70,13 @@
               {{ initTitleActionComponent(action) }}
             </div>
           </div>
+          <v-icon
+            v-exo-tooltip.bottom="$t('exoplatform.chat.add.new.discussion')"
+            v-show="chatDiscussionFeatureEnabled && !showSearch && !selectedContact"
+            class="my-auto"
+            @click="openCreateRoomDrower">
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]:
```suggestion
            @click="openQuickCreateChatDiscussionDrawer">
```

File: services/src/main/resources/conf/portal/configuration.xml [hunk 5]
```diff
@@ -34,7 +34,17 @@
           </value-param>
         </init-params>
     </component>
-
+    <component>
```
Comment (azayati) [inline services/src/main/resources/conf/portal/configuration.xml]: keep this in a separated PR

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 6]
```diff
@@ -70,6 +70,13 @@
               {{ initTitleActionComponent(action) }}
             </div>
           </div>
+          <v-icon
+            v-exo-tooltip.bottom="$t('exoplatform.chat.add.new.discussion')"
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]:
```suggestion
            v-exo-tooltip.bottom="$t('exoplatform.chat.quick.create.discussion')"
```

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: services/src/main/resources/conf/portal/configuration.xml [hunk 7]
```diff
@@ -34,7 +34,7 @@
           </value-param>
         </init-params>
     </component>
-
+
```
Comment (azayati) [inline services/src/main/resources/conf/portal/configuration.xml]: revert this change

**Review by azayati — CHANGES_REQUESTED** [review]

File: application/src/main/webapp/vue-app/chatConstants.js (hunk 1 above)
Comment (sofyenne) [reply application/src/main/webapp/vue-app/chatConstants.js]: done !

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue (hunk 2 above)
Comment (sofyenne) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]: done

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue (hunk 3 above)
Comment (sofyenne) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]: done !

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue (hunk 4 above)
Comment (sofyenne) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]: done !

File: services/src/main/resources/conf/portal/configuration.xml (hunk 5 above)
Comment (sofyenne) [reply services/src/main/resources/conf/portal/configuration.xml]: done !

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue (hunk 6 above)
Comment (sofyenne) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]: done !

File: services/src/main/resources/conf/portal/configuration.xml (hunk 7 above)
Comment (sofyenne) [reply services/src/main/resources/conf/portal/configuration.xml]: reverted !

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 8]
```diff
@@ -228,6 +236,8 @@ export default {
     document.addEventListener(chatConstants.ACTION_ROOM_OPEN_CHAT, this.openRoom);
     chatWebStorage.registreEventListener();
     chatWebSocket.registreEventListener();
+    this.$featureService.isFeatureEnabled('quickCreateChatDiscussion')
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]: put this in the other PR

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]
LGTM
